.github/workflows/checkBranch.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Should it be placed under `/workflows/` directory? If we would add more scripts here, then we will end up with a mess. Try to group them, use the same rule as for source files `.kt` or `.java`

Reply:
Just add your proposal and we will see ;)

Reply:
Ok, I added 😄